resources/ext/ezbake.conf

```diff
@@ -2,8 +2,8 @@ ezbake: {
   pe: {}
   foss: {
     redhat: { dependencies: ["puppet >= 3.7.3", "puppet < 5.0.0"],
-              postinst: ["/usr/bin/puppetdb ssl-setup"] },
-    debian: { dependencies: ["puppet (>= 3.7.3-1puppetlabs1)", "puppet (<< 5.0.0-1puppetlabs1)"],
-              postinst: ["/usr/bin/puppetdb ssl-setup"] }
+              postinst: ["PATH=/opt/puppetlabs/bin:/usr/bin:$PATH puppetdb ssl-setup"] },
```

Comment:
So you're concern is about the pathing to `puppet` specifically, not puppetdb binaries per se. So this stuff works I'm guessing: https://github.com/puppetlabs/puppetdb/blob/master/resources/ext/cli/ssl-setup#L242-L246